mkosi/distributions/opensuse.py

```diff
@@ -51,6 +52,17 @@ def install_packages(cls, state: MkosiState, packages: Sequence[str], apivfs: bo
     def remove_packages(cls, state: MkosiState, packages: Sequence[str]) -> None:
         invoke_dnf(state, "remove", packages)
 
+    @staticmethod
+    def architecture(arch: Architecture) -> str:
+        a = {
+            Architecture.x86_64 : "x86_64",
```

Comment:
suse supports the same as fedora, give or take

Reply:
Yes but it'll require different mirrors or something like that which I'll leave for suse folks to figure out.

Reply:
No, it's all on the same download server

Reply:
http://download.opensuse.org/tumbleweed/repo/oss/

I don't see anything other than x86_64 here. The ports are there but they will require more changes so I'd like to leave that for another PR.